components/UploadForm.vue

```diff
@@ -1,44 +1,141 @@
 <template>
-  <div class="mx-auto">
-    <img v-if="isImage" class="max-w-md" :src="fileData" />
-    <Button
-      v-if="exifInfo"
-      :text="$t('UploadForm.view.file.button')"
-      preset="outline"
-      @click="isOpenFileInfoDialog = true"
-    />
-    <form @submit.prevent="onSubmit">
-      <input
-        v-if="!fileData"
-        v-model="ipfsURL"
-        :placeholder="$t('UploadForm.url.placeholder')"
-        @change="onEnterURL"
-      />
-      <input
-        ref="imageFile"
-        accept="image/*"
-        type="file"
-        @change="onFileUpload"
-      />
-      <button class="block" type="submit">
-        {{ $t('UploadForm.button') }}
-      </button>
-    </form>
-
-    <Dialog
-      v-model="isOpenFileInfoDialog"
-      :has-padding="false"
-      preset="custom"
-    >
-      <MetadataCard :data="exifInfo" />
-    </Dialog>
+  <div>
+    <div class="flex flex-col mx-auto mt-[40px] w-min">
+      <Button
+        :to="localeLocation({ name: 'index' })"
+        preset="plain"
+        tag="div"
+        :text="$t('UploadForm.button.back')"
+        class="text-dark-gray"
+      >
+        <template #prepend>
+          <IconArrowLeft />
+        </template>
+      </Button>
+      <Card :has-padding="false" set-padding-classes="p-[32px]">
```

Comment:
Can override it with this?
```suggestion
      <Card class="p-[32px]" :has-padding="false">
```